### ISSUE TITLE
Repel node labels in forceNetwork

### ISSUE THREAD

**Issue by julijamell** [christophergandrud/networkD3#259]
Is it possible to have node labels repel each other to avoid label overlap?

**Comment by cjyetman** [comment]
No, not currently. And it’s not something likely to be added because of the difficulty in achieving it.

**Comment by k-blenman** [comment]
May we revisit this request. Repelling nodes seems to work fine. Can we treat text labels like nodes (i.e., like objects - maybe with a bounding box) and repel them in the same fashion.

**Comment by cjyetman** [comment]
> May we revisit this request. Repelling nodes seems to work fine. Can we treat text labels like nodes (i.e., like objects - maybe with a bounding box) and repel them in the same fashion.

This package/repo has been basically dormant for about 7 years other than some minor documentation fixes done last year to prevent it from being removed from CRAN, so I would not expect any new features, nor even fixes honestly.

**Comment by k-blenman** [comment]
@cjyetman. Understood. Thank you.
